ygen/directory.go

```diff
@@ -72,6 +72,10 @@ type YangListAttr struct {
 	// keyElems is a slice containing the pointers to yang.Entry structs that
 	// make up the list key.
 	KeyElems []*yang.Entry
+	// ListKeyYANGNames is the ordered list of YANG names specified in the
+	// YANG list per Section 7.8.2 of RFC6020. Rely on this
```

Comment:
```suggestion
	// YANG list per Section 7.8.2 of RFC6020. ygot relies on this ordering
```

Is this the right suggestion? Alternatively, do we mean that the consumer of the IR can rely on it?